packages/connect-common/files/firmware/t3t1/releases.json

```diff
@@ -1,4 +1,20 @@
 [
+    {
+        "required": false,
+        "version": [2, 8, 6],
+        "min_bridge_version": [2, 0, 7],
+        "min_bootloader_version": [2, 1, 6],
+        "min_firmware_version": [2, 7, 2],
+        "bootloader_version": [2, 1, 9],
+        "firmware_revision": "17f2a3256b2631823c2bad1855fab61b27985fd2",
+        "translations": ["cs-CZ", "de-DE", "es-ES", "fr-FR", "it-IT", "pt-BR"],
+        "url": "data/firmware/t3t1/trezor-t3t1-2.8.6.bin",
```

Comment:
The `data/` part of the path is only relevant for the data repo, should be removed.